remotes/docker/registry_test.go

```diff
@@ -52,8 +52,10 @@ func TestMatchLocalhost(t *testing.T) {
 		match bool
 	}{
 		{"", false},
-		{"127.1.1.1", false},
```

Comment:
@dmcgowan Was there a specific reason to return false for 127.1.1.1 ? Is there any security concern?